messages/TypescriptEslintStrategy.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	"FileNotIncludedByTsConfig": "'%s' does not reside in a location that is included by your %s 'include' attribute.",
+	"MissingTsConfigFromCwd": "Unable to find '%s' in current directory '%s'. Please change to a directory that contains '%s' or specify it using the --tsconfig flag",
+	"InvalidNameTsConfigFromOptions": "File '%s' specified by the --tsconfig flag must be named '%s'",
+	"NotAFileTsConfigFromOptions": "Unable to find '%s' at location '%s' specified by the 'tsconfig' option",
```

Comment:
This error is thrown by the engine, but is specified in the CLI. I removed `flag` and went with generic `option` since this could be specified via another mechanism in the future.